### ISSUE TITLE
BUG: `sample_times` action argument is not parsed correctly

### ISSUE THREAD

**Issue by jonrkarr** [RuleWorld/PyBioNetGen#16]
For example
```
simulate({t_start=>0,sample_times=>[0,1000000],output_step_interval=>1,max_sim_steps=>1,method=>"ssa"});
```

is parsed to 
```
{'t_start': '0', 'sample_times': '[0', 'output_step_interval': '1', 'max_sim_steps': '1', 'method': '"ssa"'}
```

**Comment by ASinanSaglam** [comment]
This and #15 will be resolved shortly, will update very soon.

**Comment by ASinanSaglam** [comment]
Should be fixed with 0.4.6 release
